### ISSUE TITLE
After updating from 1.7.1 to 1.8.1 Channel list - error while getting the channel list - Got SIGTERM, shutting down Homebridge...

### ISSUE THREAD

**Issue by DJay-X** [merdok/homebridge-webos-tv#242]
After updating from 1.7.1 to 1.8.1 Channel list - error while getting the channel list - Got SIGTERM, shutting down Homebridge...

Going back to 1.7.1 and everything is working again.
homebridge@1.1.0 | node@12.16.3 | npm 6.14.5

LG 55EG9209-ZA | SW: 04.26.00 | webOS 2.2.3-1351

I'm not using/having channel list because the tv is not connected to any cable provider.

```
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]: [2020-5-25 10:14:30] [LG webOS TV] [LG webOS TV] Channel list - error while getting the channel list
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]: [2020-5-25 10:14:30] [LG webOS TV] [LG webOS TV] Error message: tuner channel map does not exist
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]: Error: [LG webOS TV] Failed to get channel list
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at /usr/local/lib/node_modules/homebridge-webos-tv/index.js:293:28
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at Object.callbacks.<computed> (/usr/local/lib/node_modules/homebridge-webos-tv/node_modules/lgtv2/index.js:216:25)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at WebSocketConnection.<anonymous> (/usr/local/lib/node_modules/homebridge-webos-tv/node_modules/lgtv2/index.js:146:48)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at WebSocketConnection.emit (events.js:310:20)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at WebSocketConnection.processFrame (/usr/local/lib/node_modules/homebridge-webos-tv/node_modules/websocket/lib/WebSocketConnection.js:554:26)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at /usr/local/lib/node_modules/homebridge-webos-tv/node_modules/websocket/lib/WebSocketConnection.js:323:40
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at processTicksAndRejections (internal/process/task_queues.js:79:11)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]: Error: [LG webOS TV] Failed to get channel list
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at /usr/local/lib/node_modules/homebridge-webos-tv/index.js:293:28
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at Object.callbacks.<computed> (/usr/local/lib/node_modules/homebridge-webos-tv/node_modules/lgtv2/index.js:216:25)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at WebSocketConnection.<anonymous> (/usr/local/lib/node_modules/homebridge-webos-tv/node_modules/lgtv2/index.js:146:48)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at WebSocketConnection.emit (events.js:310:20)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at WebSocketConnection.processFrame (/usr/local/lib/node_modules/homebridge-webos-tv/node_modules/websocket/lib/WebSocketConnection.js:554:26)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at /usr/local/lib/node_modules/homebridge-webos-tv/node_modules/websocket/lib/WebSocketConnection.js:323:40
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]:     at processTicksAndRejections (internal/process/task_queues.js:79:11)
Mai 25 10:14:30 raspberrypi-3BPlus homebridge[450]: [2020-5-25 10:14:30] Got SIGTERM, shutting down Homebridge...
```

```
{
            "accessory": "webostv",
            "name": "LG webOS TV",
            "ip": "xxx.xxx.xxx.xx",
            "mac": "xx:xx:xx:xx:xx:xx",
            "keyFile": "/home/pi/.homebridge/lgtvKeyFile",
            "volumeControl": "buttons",
            "mediaControl": true,
            "showInputButtons": true,
            "legacyTvService": true,
            "pollingInterval": 10,
            "inputs": [
                {
                    "appId": "maxdome",
                    "name": "Maxdome"
                },
                {
                    "appId": "com.webos.app.hdmi1",
                    "name": "Hauptbildschirm"
                }
            ],
            "notificationButtons": [
                "Backofen vorgeheizt.",
                "Wasserkocher fertig.",
                "Haustür klingelt.",
                "Luftqualität in der Wohnung schlecht. Bitte lüften."
            ]
        },
```

**Comment by merdok** [comment]
Oh, right. In that case i should not throw an error, just log that. I will fix that later today and submit a fix! Thanks for reporting!

**Comment by DJay-X** [comment]
And definitely not crash the entire Homebridge. 😅
Thanks for your great work and friendly support.

**Comment by merdok** [comment]
Yeah, sorry. Well I can only test stuff on my tv, but out there users has numerous other different configurations so I unfortunately I cannot always catch all the issues, especially when I do some major changes...

**Comment by merdok** [comment]
Should be fixed in version 1.8.2
